smoderp2d/core/general.py

```diff
@@ -208,6 +208,8 @@ class Globals:
     extraOut = None
     # stream magic number
     streams_flow_inc = 1000
+    # slope width 
+    slope_width = None
```

Comment:
```
slope_width = 1
```

Reply:
I set it here to `None` because it is then used in the loop in `base provider` to assign a new value if the `slope_width` is a key in the dictionary `data` (bellow in this comment). In my solution the dictionary `data` containers the key `slope_width` only in case of the nogis provider. Other providers does not have it. 

```
smoderp2d/providers/base/__init__.py
236         for item in data.keys():
237             if hasattr(Globals, item):
238                 if getattr(Globals, item) is None:
239                     setattr(Globals, item, data[item])
```

Reply:
However, if your solution will work in the all providers, lets use it.